### PR TITLE
Workflows - Move variables and params info to their own top-level page

### PR DESCRIPTION
**Pull request by neko-dd**
### What does this PR do? What is the motivation?

Consolidates variable and parameter info into one page at the top level.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/service_management/workflows/variables.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+1. Add a parameter name, value, and data type for the parameter.
+1. Optionally, add a default value for the parameter. If you add a default value, the parameter is optional at runtime.
+
+The **Output Parameters** section displays the names of all existing output parameters together with a counter.
+
+For an example of how to use output parameters to pass information between Workflows and App Builder, see [return workflow results to an app][6].
+
+## Custom variables
+
+To set a custom mutable workflow variable, use the [Set variable][1] action. You can use this action to declare, update, and access variables throughout your workflow, which allows you to perform more complex workflow operations. For example:
```
Comment (hestonhoffman) [inline content/en/service_management/workflows/variables.md:97]:
```suggestion
To set a mutable workflow variable, use the [Set variable][1] action. You can use this action to declare, update, and access custom variables throughout your workflow, which allows you to perform more complex workflow operations. For example:
```

File: content/en/service_management/workflows/variables.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+- _Handling API pagination_: API requests sometimes require you to keep track of a page token or offset.
+- _Handling lists_: You can use a variable to initialize an array and perform actions like map and reduce.
+- _Iteration_: Variables allow you to manipulate and store data inside a [for loop][2]. You can then use that data in the rest of the workflow.
+
+### Set a variable
```
Comment (hestonhoffman) [inline content/en/service_management/workflows/variables.md:102]:
```suggestion
### Set a custom variable
```

File: content/en/service_management/workflows/variables.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+{{< img src="service_management/workflows/input-parameter3.png" alt="Hover over a counter to see which steps are using the parameter." style="width:60%;">}}
+
+You can add an implicit input parameter (a parameter that doesn't already exist in the workflow) by typing it into a workflow step using the `{{ Trigger.<parameter name> }}` syntax. The next time you save the workflow, a dialog appears allowing you to convert the parameter to an explicit parameter. For more information on triggering workflows, see [Trigger a workflow][5].
+
+If you're looking for an existing input parameter, start typing `{{ Trigger.` to see if it appears as a suggestion. Alternatively, consult the [Context Variables](#context-variables) tab for a list of available parameters.
+
+## Output parameters
+
+You can pass the result of a workflow using output parameters.
```
Comment (hestonhoffman) [inline content/en/service_management/workflows/variables.md:83]:
```suggestion
Output parameters allow you to access the result of a workflow. This is useful when you want to pass the result of a workflow to another workflow or an App Builder app.
```

File: content/en/service_management/workflows/variables.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+To add an output parameter:
+1. Click on the workflow canvas.
+1. Click the **+** icon next to **Output Parameters**.
+1. Add a parameter name, value, and data type for the parameter.
+1. Optionally, add a default value for the parameter. If you add a default value, the parameter is optional at runtime.
+
+The **Output Parameters** section displays the names of all existing output parameters together with a counter.
+
```
Comment (hestonhoffman) [inline content/en/service_management/workflows/variables.md:92]:
```suggestion
For information on using output parameters with workflows, see [Trigger a workflow from a workflow][7].
```
Need to add a small section to this page for that. I can follow up with a commit

File: content/en/service_management/workflows/variables.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+{{< partial name="whats-next/whats-next.html" >}}
+
+<br>Do you have questions or feedback? Join the **#workflows** channel on the [Datadog Community Slack][3].
+
+[1]: https://app.datadoghq.com/workflow/action-catalog#/com.datadoghq.core/com.datadoghq.core.setVariable
+[2]: /service_management/workflows/actions/flow_control#for-loop
+[3]: https://datadoghq.slack.com/
+[4]: https://handlebarsjs.com/guide/expressions.html#expressions
+[5]: /service_management/workflows/trigger
+[6]: /service_management/app_builder/queries/#return-workflow-results-to-an-app
```
Comment (hestonhoffman) [inline content/en/service_management/workflows/variables.md:148]:
```suggestion
[6]: /service_management/app_builder/queries/#return-workflow-results-to-an-app
[7]: /service_management/workflows/trigger/#trigger-a-workflow-from-a-workflow
```

File: content/en/service_management/workflows/build.md [hunk 6]
```diff
@@ -87,68 +87,6 @@ Scheduled and triggered workflows don't trigger automatically until you've publi
 
 Published workflows accrue costs based on workflow executions. For more information, see the [Datadog Pricing page][4].
 
-## Context variables
```
Comment (ambertunnell) [inline content/en/service_management/workflows/build.md:90]: Should we link out to the new variables doc in this doc now?

**Pull request merged**